queue/bytes_queue_test.go

```diff
@@ -71,7 +100,7 @@ func TestAllocateAdditionalSpaceForInsufficientFreeFragmentedSpaceWhereHeadIsBef
 	t.Parallel()
 
 	// given
-	queue := NewBytesQueue(25, false)
+	queue := NewBytesQueue(25, true)
```

Comment:
Output from bench tests is cleaner when verbose mode in unit tests is disabled as bench tests also run unit tests.